patches/node/fix_set_uptime_offset_in_correct_init_method.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Micha Hanselmann <deermichel@github.com>
+Date: Wed, 24 Jul 2019 11:44:25 -0700
+Subject: fix: set uptime offset in correct init method
```

Comment:
Can we add a note to this patch explaining when it can be removed and linking to the upstream nodejs PR.  At some point this patch will disappear and that context will help 👍